packages/react-native-editor/sass-transformer.js

```diff
@@ -117,7 +122,12 @@ function transform( src, filename, options ) {
 						path.resolve( path.dirname( filename ), urlPath.dir )
 					); // Add the file's dir to the search array.
 				}
-				const f = findVariant( urlPath.name, exts, incPaths );
+				const f = findVariant(
+					urlPath.name,
+					exts,
+					incPaths,
+					options.projectRoot
```

Comment:
How does `projectRoot` arrive here? Is it automatically provided from the Metro config?

Reply:
For what I could see yeah even if you don't manually set a `projectRoot` it'll use the value React Native detects.